conf/muchos.props.example

```diff
@@ -129,6 +129,31 @@ metrics_drive_root = var-data
 # Optional proxy VM. If not set, the first node of the cluster will be selected as the proxy.
 azure_proxy_host =
 location = westus2
+# Enable ADLS Gen2 storage configuration. Muchos parameters instance_volumes_input, instance_volumes_preferred & adls_storage_type is not required if use_adlsg2 is false.
+use_adlsg2 = False
+# Storage accounts can be auto generated or manually specified. "|" is used as separator between manual and auto generated storage account names and must be specified
+# Manual and Auto generated names are mutually exclusive
+#
+# Specifying storage accounts manually:
+#    |abfss://<container-name>@<storage-account-name>.<domain-name>/<folder-name>". Use comma to specify multiple entries
+#    Example:|abfss://accumulodata@shnawastore1.dfs.core.windows.net/accumulo,abfss://accumulodata@shnawastore2.dfs.core.windows.net/accumulo
+#
+# Specifying auto-generated storage accounts:
+#   <Number-of-Storage-Accounts>,<domain-name>|
+#   Example: 3,dfs.core.windows.net|
+instance_volumes_input = 1,dfs.core.windows.net|
+# Do not update "instance_volumes_preferred", it will be populated dynamically during launch phase of muchos
+instance_volumes_preferred =
+# Type of storage for ADLS Gen2 storage accounts
+adls_storage_type = Standard_LRS
+# Specify user assigned identity name. "{{ vmss_name }}-ua-msi"  will be created if value is not provided
+user_assigned_identity =
+# Do not update "azure_tenant_id", it will be populated dynamically during launch phase of muchos
+azure_tenant_id =
```

Comment:
The tenant_id is required for setup phase of  cluster and sourced from muchos.props during the setup

Reply:
I realize its automatically set, I was just wondering if it could be left out of the example file and still automatically set.

Reply:
Since the parameters are part of Azure group, the place holder helps. Are you OK leaving the parameters in example file?

Reply:
Im ok leaving it.  I was thinking including it makes the file harder for people read.  However, if a users file is going to be modified its probably best to leave it and have a comment.  

This would be an entirely different issue/PR, but maybe it would be better to put these automatically generated props in another file.  That way muchos.props is only modified by the user.